### DISCUSSION TITLE
Onboarding PR not re-created

### DISCUSSION THREAD

**Discussion by ClementParis016** [renovatebot/renovate#11119] [Request Help] [answered]
Hello 👋🏻 

I have a repo where Renovate's onboarding PR has been closed and I would like to get it back.

I followed [the instructions for it to re-create the PR](https://github.com/renovatebot/config-help/issues/54):
1. close the PR
2. rename it
3. made sure we don't have a config file remaining

But it's been 8 days now and the PR hasn't been re-created by Renovate 🤔 

What should I do?

**Comment by rarkins** [comment, marked as answer]
Please let me know the repository name or any past job ID if you have one and I'll restart it manually

**Reply by ClementParis016** [reply]
Oh nice, you can do that :) It's at Spendesk/service-template (private).

But I found something interesting: this repository is a GitHub template and when I click the link to view the job logs in the old onboarding PR it takes me to the jobs of an other repository which has been created from this template. So there may be something weird going on with templates?

**Reply by rarkins** [reply]
I think you should see an onboarding PR now

**Reply by ClementParis016** [reply]
I can see it, thank you very much 🙏🏻
